examples/index.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <!-- <script type="text/javascript" src="../dist/index.umd.js"></script> -->
```

Comment:
I would rather load it locally than from unpkg

Reply:
This is so that we can turn on GitHub Pages and add a URL to this demo page.

Reply:
Ah good point!